README.md

```diff
@@ -1,53 +1,144 @@
-7. Migrate the database with rake db:migrate
-8. Access and write to a model from a Rails console
-9. Read a model spec for an instance method
-10. Describe how controllers and views have access to our model constant by convention and usage 
-11. Read a database.yml and describe how it connects to different environment databases
+ActiveRecord is the built in ORM that Rails utilizes to manage the model aspects of an application. What is an ORM? An ORM is an Object Relational Mapping system, essentially this is the module that enables your application to manage data in a method driven structure. This means that you are able to run queries, add records, and perform all of the traditional database processes by leveraging methods as opposed to writing SQL manually. For example, below is the traditional way that we would query a database of 'posts' using SQL:
 
+```"SELECT * FROM Posts"```
 
-## Notes
+Compared with leveraging ActiveRecord:
 
-Build instance methods that extend a basic AR class (remind them that as a model AR classes interact with our database but can still function as normal classes and provide abstractions ontop of the data. They've done this previously in sinatra so they are familiar with AR)
+```ruby
+Post.all
+```
 
-AR models provide an abstraction for our database and persistance and lots of logic and functionality relating to the data our models provide to our application.
+By using ActiveRecord you are also able to perform advanced query tasks, such as method chaining and scoping, which typically requires less code and makes for a more readable query.
 
-Our database might have a first_name and last_name column, which AR provides access to with first_name, and last_name readers/writers, but if we wanted to print a full_name, we'd build a full_name instance method that combines the first_name and last_name, so our models aren't just data, but also we build these sort of data helper methods.
+## ActiveRecord Models
 
-First we need a table, so let's make a migration (they've done this before!)
+So if we have a database table, why do we need a model file? By using model files we are able to create an organized layer of abstraction for our data. An important item to remember is that at the end of the day the model file is a Ruby class. It will typically inherit from the ActiveRecord::Base class, which means that it has access to a number of methods that assist in working with the database. However, you can treat it like a regular Ruby class, allowing you to create methods, data attributes, and everything else that you would want to do in a class file.
 
-001_create_posts.rb
+A typical model file will contain code such as, but not limited to the following:
 
-we dont need to go too much into Migration syntax as they mostly learned this already or this readme can assume they are familiar with AR Basics such as migrations and stuff. but we'll have the migration code and link to migration docs for more information.
+* Custom scopes
```

Comment:
Sounds good, I'll put links for each item to the documentation and also add a disclaimer so they aren't discouraged about not hearing about the topics yet.